code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -135,6 +135,7 @@
 				/obj/item/caution = 10,
 				/obj/item/mod/construction/broken_core = 4,
 				/obj/effect/spawner/random_spawners/mod/maint = 10,
+				/obj/item/bikehorn/bwoinkhorn = 5,
```

Comment:
I would personally make it a prize counter item instead of maints loot that doesnt even has the base bikehorn on it (just my opinion).